hubble/cmd/common/config/flags.go

```diff
@@ -28,6 +28,11 @@ const (
 	KeyBasicAuthPassword = "basic-auth-password"  // string
 	KeyTimeout           = "timeout"              // time.Duration
 	KeyRequestTimeout    = "request-timeout"      // time.Duration
+	KeyAutoPortForward   = "auto-port-forward"    // bool
+	KeyPortForward       = "port-forward"         // uint16
+	KeyK8sContextName    = "k8s-context"          // string
+	KeyK8sNamespace      = "k8s-namespace"        // string
+	KeyK8sKubeconfig     = "k8s-kubeconfig"       // string
```

Comment:
If you have a suggestion for alternative names, don't hesitate.

I also though of grouping port-forward flags together similar to `--tls` like:
```
      --port-forward                  Automatically forward the relay port to the local machine. Analoguous to running: 'cilium hubble port-forward'.
      --port-forward-port uint16      Local port to forward to. 0 will select a random port. This option is only considered when --auto-port-forward is set. (default 4245)
```

Reply:
I like `--port-forward-port` more than `--port-forward` for configuring the port. Otherwise I think the rest is fine. 

If I was to nit-pick`--k8s-kubeconfig` could probably be just `--kubeconfig` since we don't have to worry about the potential conflict if we add config/contexts for hubble CLI. I could also argue `--kube-context`/`--kube-namespace` instead of `--k8s-context`/`--k8s-namespace` which is the naming scheme argocd CLI uses, but I don't have strong feelings about that. Ill defer to you on those.

Reply:
I like your suggestions, will update, thanks